pkg/datapath/iptables/iptables.go

```diff
@@ -1035,12 +1046,56 @@ func (m *IptablesManager) installForwardChainRulesIpX(prog iptablesInterface, if
 	return nil
 }
 
+// AddToNodeIpset adds an IP address to the ipset for cluster nodes. It creates
+// the ipset if it doesn't already exist and doesn't error if either the ipset
+// or the IP already exist.
+func AddToNodeIpset(nodeIP net.IP) {
+	ciliumNodeIpset := ciliumNodeIpsetV4
+	if ip.IsIPv6(nodeIP) {
+		ciliumNodeIpset = ciliumNodeIpsetV6
+	}
+	log.WithField(logfields.IPAddr, nodeIP.String()).Debugf("Adding IP to %s ipset", ciliumNodeIpset)
+	createIpset(ciliumNodeIpset, ip.IsIPv6(nodeIP))
+	progArgs := []string{"add", ciliumNodeIpset, nodeIP.String(), "-exist"}
+	ipset.runProg(progArgs, false)
+}
+
+// RemoveFromBodeIpset removes an IP address from the ipset for cluster nodes.
+func RemoveFromNodeIpset(nodeIP net.IP) {
+	ciliumNodeIpset := ciliumNodeIpsetV4
+	if ip.IsIPv6(nodeIP) {
+		ciliumNodeIpset = ciliumNodeIpsetV6
+	}
+	log.WithField(logfields.IPAddr, nodeIP.String()).Debugf("Removing IP to %s ipset", ciliumNodeIpset)
```

Comment:
```suggestion
	log.WithField(logfields.IPAddr, nodeIP.String()).Debugf("Removing IP %s from %s ipset", nodeIP, ciliumNodeIpset)
```

Reply:
The IP address is already logged with the `WithField(...)` :wink:

Reply:
Doh. :D

Reply:
Only suggestion then would be `to` -> `from`.

Reply:
Yep, I fixed that :slightly_smiling_face: